docs/guides/integration-mobile-client-server.md

```diff
@@ -0,0 +1,361 @@
+---
+title: UID2 Client-Server Integration Guide for Mobile
+sidebar_label: Client-Server Integration for Mobile
+pagination_label: UID2 Client-Server Integration Guide for Mobile
+description: Setting up a mobile integration with token generate on server and refresh on client.
+hide_table_of_contents: false
+sidebar_position: 04
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Link from '@docusaurus/Link';
+import ReduceLatency from '/docs/snippets/_sdk-reduce-latency.mdx';
+import EnableLogging from '/docs/snippets/_mobile-docs-enable-logging.mdx';
+import GMAIMA_Plugins from '/docs/snippets/_mobile_docs_gmaima-plugin-gss.mdx';
+
+# UID2 Client-Server Integration Guide for Mobile
+
+This guide is intended for mobile app publishers who want to integrate with UID2 by generating UID2 tokens server-side via either a public or Private Operator and then passing the tokens and user identities into their mobile apps, which will in turn pass the tokens for bid stream use.  
```

Comment:
why _public_/_Private_ casing is different?

Reply:
Fixed (to initial caps), thx.